RecoHGCal/TICL/plugins/TrackstersPCA.cc

```diff
@@ -52,10 +58,24 @@ void ticl::assignPCAtoTracksters(std::vector<Trackster> &tracksters,
       fillPoint(layerClusters[trackster.vertices(i)], weight);
       for (size_t j = 0; j < 3; ++j)
         barycenter[j] += point[j];
+
+      // Also compute timing
+      auto before = usedLC.size();
+      usedLC.insert(trackster.vertices(i));
```

Comment:
Maybe we could use the returned value of the `insert` method to check if the cluster was not already present?
That, IMHO, would make the code cleaner to read and understand.

Reply:
Done:
https://github.com/cms-sw/cmssw/pull/32046/commits/e23de8a5c411e3cc99ddda39f11555979174d8fa?file-filters%5B%5D=.cc#diff-c2b0a484f5de9311737b9441259b0e5f49ddb15fe040d8e90e8e03d49fc6b170